app/models/action_page.rb

```diff
@@ -36,6 +54,24 @@ class ActionPage < ActiveRecord::Base
 
   scope :categorized, ->(category) { joins(:category).where(categories: { title: category }) }
 
+  def self.type(*types)
```

Comment:
This looks like it was written to support multiple types, but I'm only able to filter on a single type because the form utilizes a radio button element. Is that intentional?

Reply:
Sort of. I was experimenting with checkbox vs. radio behavior for the action type filter. The UX felt better with radios, but I kept this method as it was when it supported both.

Reply:
Cool, that sounds fine to me. I thought the radio worked well from a UX perspective too.